verilog/formatting/tree_unwrapper.cc

```diff
@@ -1162,6 +1162,20 @@ static bool PartitionIsCloseBrace(const TokenPartitionTree& partition) {
   return token_enum == '}';
 }
 
+static bool PartitionEndsWithOpenBrace(const TokenPartitionTree& partition) {
+  const auto ftokens = partition.Value().TokensRange();
+  if (ftokens.empty()) return false;
+  const auto token_enum = ftokens.back().TokenEnum();
+  return token_enum == '{';
+}
+
+static bool PartitionStartsWithCloseBrace(const TokenPartitionTree& partition) {
+  const auto ftokens = partition.Value().TokensRange();
+  if (ftokens.empty()) return false;
+  const auto token_enum = ftokens.front().TokenEnum();
+  return (token_enum == '}');
```

Comment:
Parenthesis around here is this is inconsistent with line 1169 (which does not use parenthesis; preferred).

Reply:
Fixed